pkg/k8sutils/k8sutils.go

```diff
@@ -287,3 +290,36 @@ func GetConfigMap(ctx context.Context, k8scient client.Client, namespace, name s
 
 	return &configMap, nil
 }
+
+var (
+	KUBE_MAJOR_VERSION string
+	KUBE_MINOR_VERSION string
+)
+
+// GetKubernetesVersion get kubernetes version. It should not be executed concurrently.
+// The global variable KUBE_MAJOR_VERSION and KUBE_MINOR_VERSION will be set.
+func GetKubernetesVersion() error {
+	config, err := clientcmd.BuildConfigFromFlags("", "~/.kube/config")
```

Comment:
does this mean that the operator pod needs to be configured with .kube/config?

Reply:
It will first check `~/.kube/config`. If it does do not exist, operator will use the the config of starrocks service account.

Reply:
shouldn't assume that the .kube/config are never available in a k8s deployment env?